src/components/YesNoContest.tsx

```diff
@@ -40,84 +47,73 @@ interface Props {
   updateVote: UpdateVoteFunction
 }
 
-interface State {
-  isScrollable: boolean
-  isScrollAtBottom: boolean
-  isScrollAtTop: boolean
-  overvoteSelection: Optional<YesOrNo>
-}
-
-const initialState: State = {
-  isScrollable: false,
-  isScrollAtBottom: true,
-  isScrollAtTop: true,
-  overvoteSelection: undefined,
-}
+const YesNoContest = ({ contest, vote, updateVote }: Props) => {
+  const context = useContext(BallotContext)
+  const scrollContainer = useRef<HTMLDivElement>(null) // eslint-disable-line no-restricted-syntax
 
-export default class YesNoContest extends React.Component<Props, State> {
-  public context!: React.ContextType<typeof BallotContext>
-  public state = initialState
-  private scrollContainer = React.createRef<HTMLDivElement>()
+  const [isScrollable, setIsScrollable] = useState(false)
+  const [isScrollAtBottom, setIsScrollAtBottom] = useState(true)
+  const [isScrollAtTop, setIsScrollAtTop] = useState(true)
+  const [overvoteSelection, setOvervoteSelection] = useState<Optional<YesOrNo>>(
+    undefined
+  )
```

Comment:
You can simplify this by leaving out the initial value:

```suggestion
  const [overvoteSelection, setOvervoteSelection] = useState<YesOrNo>()
```

Reply:
thx. done.